src/python/pants/backend/jvm/subsystems/zinc.py

```diff
@@ -32,7 +32,9 @@
 from pants.util.memo import memoized_method, memoized_property
 
 
-_ZINC_COMPILER_VERSION = '0.0.9'
+# TODO: To use this with the nailgun strategy, will need a publish of `rsc_2.12`: this will
+# block landing this.
```

Comment:
This can be removed now.

Reply:
...but since this is green, I'm going to followup to remove it with the fix for #7850.